lib/asyncapiSchemaFormatParser.js

```diff
@@ -47,12 +47,15 @@ function getMimeTypes() {
     'application/vnd.aai.asyncapi;version=2.0.0',
     'application/vnd.aai.asyncapi;version=2.1.0',
     'application/vnd.aai.asyncapi;version=2.2.0',
+    'application/vnd.aai.asyncapi;version=2.3.0',
     'application/vnd.aai.asyncapi+json;version=2.0.0',
     'application/vnd.aai.asyncapi+json;version=2.1.0',
     'application/vnd.aai.asyncapi+json;version=2.2.0',
+    'application/vnd.aai.asyncapi+json;version=2.3.0',
     'application/vnd.aai.asyncapi+yaml;version=2.0.0',
     'application/vnd.aai.asyncapi+yaml;version=2.1.0',
     'application/vnd.aai.asyncapi+yaml;version=2.2.0',
+    'application/vnd.aai.asyncapi+yaml;version=2.3.0',
```

Comment:
At this point, I'm already wondering if we should refactor this to something like:

```suggestion
    ...['2.0.0', '2.1.0', '2.2.0', '2.3.0'].map((version) => {
      [
        `application/vnd.aai.asyncapi;version=${version}`,
        `application/vnd.aai.asyncapi+json;version=${version}`,
        `application/vnd.aai.asyncapi+yaml;version=${version}`,
      ]
    }).flat()
```

What do you think?

Reply:
@fmvilas  Definitely makes sense as then it is just a matter of adding new version to the list once, no copy/paste.

Just small improvement, missing return and generic mime types:
```
function getMimeTypes() {
  return [
    ...['2.0.0', '2.1.0', '2.2.0', '2.3.0'].map((version) => {
      return [
        `application/vnd.aai.asyncapi;version=${version}`,
        `application/vnd.aai.asyncapi+json;version=${version}`,
        `application/vnd.aai.asyncapi+yaml;version=${version}`,
      ]
    }).flat().concat(['application/schema;version=draft-07', 'application/schema+json;version=draft-07', 'application/schema+yaml;version=draft-07'])
  ];
}
```